apps/gitness/src/components/breadcrumbs/breadcrumbs-utils.ts

```diff
@@ -11,58 +11,76 @@ const repoRoutePathNameMap = [
   { path: '/settings/general', label: 'Settings' }
 ]
 
-export function getBreadcrumbMatchers(pathname: string) {
-  const isProjectsRoute = !!matchPath(
+const matchPathUtil = ({ path, end, isMFE }: { path: string; end: boolean; isMFE: boolean }, pathname: string) => {
+  if (!isMFE) {
+    return matchPath({ path, end }, pathname)
+  }
+
+  const pathToMatch = path.replace('/:spaceId', '')
+
+  const isMatch = matchPath({ path: pathToMatch, end }, pathname)
+
+  return isMatch
+}
+
+export function getBreadcrumbMatchers(pathname: string, isMFE: boolean) {
```

Comment:
is this even used anymore? I thought this file was part of the old breadcrumbs. we should not need this anymore with the new routing mechanism.

Reply:
Agreed, this should not be required anymore.